examples/rules_lang/lang/toolchain/default_toolchain/tools/BUILD.bazel

```diff
@@ -0,0 +1,43 @@
+load("@rules_toolchains//toolchains:tool_map.bzl", "tool_map")
+
+package(default_visibility = ["//visibility:public"])
+
+alias(
+    name = "compile_tool",
+    # actual = "//lang/toolchain/default_toolchain/tools/linux_x64:compile_tool",
+    actual = select({
+        # This would normally be defined in a repo rule to allow you to fetch the compiler from the network.
+        "@rules_toolchains//tests/platforms:exec1": "//lang/toolchain/default_toolchain/tools/linux_x64:compile_tool",
+        "@rules_toolchains//tests/platforms:exec2": "//lang/toolchain/default_toolchain/tools/linux_x64:compile_tool",
+        "@rules_toolchains//tests/platforms:host": "//lang/toolchain/default_toolchain/tools/linux_x64:compile_tool",
+        "//conditions:default": "@platforms//:incompatible",
+    }),
+    # Manual because we require a transition to the exec platform.
+    # You shouldn't need this.
+    tags = ["manual"],
+)
+
+alias(
+    name = "link_tool",
+    actual = select({
+        # This would normally be defined in a repo rule to allow you to fetch the compiler from the network.
+        "@rules_toolchains//tests/platforms:exec1": "//lang/toolchain/default_toolchain/tools/linux_x64:link_tool",
+        "@rules_toolchains//tests/platforms:exec2": "//lang/toolchain/default_toolchain/tools/linux_x64:link_tool",
+        "@rules_toolchains//tests/platforms:host": "//lang/toolchain/default_toolchain/tools/linux_x64:link_tool",
+        "//conditions:default": "@platforms//:incompatible",
+    }),
+    # Manual because we require a transition to the exec platform.
+    # You shouldn't need this.
+    tags = ["manual"],
+)
+
+tool_map(
+    name = "tool_map",
+    # Manual because we require a transition to the exec platform.
+    # You shouldn't need this.
```

Comment:
Not need, but perhaps want? Otherwise you might see unnecessary fetches in cross-platform builds, right?

Reply:
I assume you're referring to when you build `//...` or something similar?